src/config/config.ts

```diff
@@ -1,17 +1,34 @@
 export interface ConfigInterface {
-  port: number;
-  dbHost: string;
-  dbUser: string;
-  dbPassword: string;
-  dbPort: number;
-  dbForceSSL: boolean;
-  a0Enabled: boolean;
-  a0Domain?: string;
-  a0Audience?: string;
-  sentryEnabled: boolean;
-  sentryDsn?: string;
-  debugLogger?: boolean;
-  testLogger?: boolean;
+  // Configuration for the http server itself
+  http: {
+    port: number;
+  };
+  // Configuration for the postgres database
+  db: {
+    host: string;
+    user: string; // For the server's own user
+    password: string; // For the server's own user
+    port: number;
+    forceSSL: boolean;
+  };
+  // Configuration for server logging
+  logger: {
+    debug: boolean; // Whether or not debug logging is enabled (does `logger.debug` do anything)
+    test: boolean; // Whether or not a special test logger is enabled (bypass weirdness with Jest)
+  }
+  // Configuration for auth0 access control
+  auth?: { // Not including this sub-object implies it is not enabled
+    domain: string;
+    audience: string;
+  };
+  // Configuration for sentry error reporting
+  sentry?: { // Not including this sub-object implies it is not enabled
+    dsn: string;
```

Comment:
shouldn't this be `dns`? 😄

Reply:
Nope. It's DSN. [Sentry Data Source Name](https://docs.sentry.io/product/sentry-basics/dsn-explainer/)